lib/analyzer/static-analyzer.ts

```diff
@@ -105,12 +115,20 @@ export async function analyze(
 
   const binaries = getBinariesHashes(archiveLayers);
 
+  const applicationScanResults: ScannedProjectCustom[] = [];
```

Comment:
consider renaming to ```applicationDependenciesScanResults``` or basically anything that helps us stop using "app" because we're not scanning applications, we're scanning application dependencies